### DISCUSSION TITLE
Do you have an estimated time for next release?

### DISCUSSION THREAD

**Discussion by apupier** [apache/rocketmq#7770] [General]
Hello,

I'm interested to know when is planned the next release?

i'm particularly interested to have this commit included https://github.com/apache/rocketmq/commit/9a5690b91abc15c76a5b4a816a2a47cb5ecfc7f1 (upgrade of snakeyaml to 2.x)

regards,

**Comment by francisoliverlee** [comment]
working on it, pls wait for a while. some has done
https://github.com/apache/rocketmq/pulls?q=Bump+snakeyaml+from+1.32+to+2.0
